projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -119,6 +119,10 @@ datajobs.notification.owner.name=Versatile Data Kit
 
 # The gitlab repository and credentials for pulling data jobs code when building their images.
 datajobs.git.url=${GIT_URL}
+datajobs.git.cc.grc=${GIT_GRC_URL}
+
+# datajobs.git.assumeIAMRole tells the control-service if the Service Account pattern should be used for AWS CodeCommit.
+datajobs.git.assumeIAMRole=${DATAJOBS_CC_AWS_ASSUME_IAM_ROLE:false}
```

Comment:
New variables would also need to be exposed and documented n https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml 

At least there's where we've tried to sort of maintain documentation and list of control service configuration.